src/main/kotlin/com/dsrm/dsrmbackend/properties/JwtProperties.kt

```diff
@@ -0,0 +1,14 @@
+package com.dsrm.dsrmbackend.properties
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import java.security.Key
+
+
+@ConfigurationProperties(prefix = "dsrm.auth.jwt")
+class JwtProperties {
+    var expirationMs = 0
+
+    var refreshExpirationMs = 0
+
+    var secretKey: Key? = null
```

Comment:
We also need to update READme with description of those values. We still miss secretKey in readme